app/utils/build-provider-asset-path.js

```diff
@@ -0,0 +1,16 @@
+import config from 'ember-get-config';
+import pathJoin from '../utils/path-join';
+
+export default function buildProviderAssetPath(providerId, assetName, isDomain) {
+    if (config.providerAssetsURL != 'local') {
+        return pathJoin(config.providerAssetsURL, providerId, assetName);
+    }
+    if (config.ASSET_SUFFIX) {
+        assetName = assetName.replace(/(\.[^\.]+)$/, function(match, $1) { return `-${config.ASSET_SUFFIX}${$1}`; });
+    }
+    const providerAssetPath = 'assets/osf-assets/files/preprints-assets';
```

Comment:
Yeah, it felt wrong hardcoding this here. If it's in config, it'd be easier to update if the assets repo structure changes, we bring assets in using a different method, etc.